app.config.js

```diff
@@ -45,6 +46,20 @@ module.exports = {
     },
     package: 'org.graasp.mobile',
     googleServicesFile: process.env.GOOGLE_SERVICES_JSON,
+    intentFilters: [
+      {
+        action: 'VIEW',
+        autoVerify: true,
+        data: [
+          {
+            scheme: "https",
+            host: "*.auth.graasp.org",
```

Comment:
Is the wildcard supposed to allow allow any domain like dev, stage and prod ? 
We use the following domains:
- `auth.dev.graasp.org`
- `auth.stage.graasp.org`
- `auth.graasp.org`

Reply:
Changed to unique `mobile.graasp.otg`